### ISSUE TITLE
regression test failure with 0.13.0 release

### ISSUE THREAD

**Issue by chenrui333** [inko-lang/inko#615]
👋 trying to build the latest release, but run into some build issue. The error log is as below:

<details>
<summary>error build log</summary>

```
  ==> /opt/homebrew/Cellar/inko/0.13.0/bin/inko run hello.inko
  hello.inko:1:11 error(invalid-syntax): Expected a top-level expression, found ':' instead
```

</details>

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/6518955982/job/17704959702
relates to Homebrew/homebrew-core#150895

**Comment by yorickpeterse** [comment]
@chenrui333 The import syntax is changed from `import foo::bar` to `import foo.bar`. This _is_ included in the release post, but it isn't published yet as I ran into some build failures for ARM64 macOS. These are fixed in `main`, and I'll be publishing 0.13.1 shortly, which should build just fine. You'll probably want to just skip 0.13.0 in favour of going straight to 0.13.1 when it's out.

**Comment by chenrui333** [comment]
> @chenrui333 The import syntax is changed from `import foo::bar` to `import foo.bar`. This _is_ included in the release post, but it isn't published yet as I ran into some build failures for ARM64 macOS. These are fixed in `main`,

thanks for the context!
